### PR TITLE
CommandBar: Fix className prop duplication

### LINKED ISSUE #6828: className prop is duplicated in DOM when applied to CommandBar
### Please provide a reproduction of the bug in a codepen:
 
https://codepen.io/anon/pen/xyyPRB

### Bug Report 
 
- __Package version(s)__: current
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes if needed
 
Requested priority: Normal
 
#### Describe the issue:

If I apply a className on CommandBar it renders the className's application in the DOM to multiple DIVs.
image

 
Line that might be the culprit where it merges the className in. Then applied to the FocusZone.
https://github.com/OfficeDev/office-ui-fabric-react/blob/4b2e44d41e1cba8d4ca5e7542d3cc66f961e0b43/packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx#L67
 
#### Actual behavior:
 className shows twice
 
#### Expected behavior:
 
className shows once on the root div.

### PR DESCRIPTION
**Pull request by JasonGore**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #6828
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Remove duplicate application of `className` prop on multiple divs.

#### Focus areas to test

Snapshots updated.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
       cacheKey: ''
     };
 
-    this._classNames = getClassNames(styles!, { theme: theme!, className });
+    this._classNames = getClassNames(styles!, { theme: theme! });
```
Comment (JasonGore) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: @micahgodbolt I'm a bit concerned if this is the right chang. I'm not sure why the `root` style is not applied to the `ResizeGroup` actual root element vs. the `FocusZone` which is a child of the `ResizeGroup`. 

This change applies `className` prop to the actual `root` element. I just want to confirm: (1) this is the right dupe to remove and (2) `root` style is applied to the correct element. Thanks

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: className should be on the root, not inside.

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: agreed, classname should always be the root visual div (I say visual as layered things have their root inside of the layer)

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: So you're both confirming `styles.root` is applied at the right level as it currently is, correct?

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: Please note `styles.root` IS applied inside and I want to make sure we're all on the same page here.

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: image

**Comment by JasonGore** [comment]
I created #6838 to resolve root styling. This PR is ready for approval.

**Pull request merged**